client/src/components/DisplayWeather/SunPosition/SunPosition.tsx

```diff
@@ -1,8 +1,12 @@
-import { useState } from 'react'
+import { useCallback, useState } from 'react'
 import './SunPosition.css'
 
   //TODO - JSON response coming in with nested data needs a type... 
   const SunPosition = (props: any) => {
+
+    const setDay = useCallback((x: boolean) => {
+      props.isDay(x)
+    }, [props.isDay])
   
   const { DateTime } = require("luxon");
   
```

Comment:
I'd suggest using setDay(true); once before the case statement so it's always day time